pages/system/card-tabs.js

```diff
@@ -72,19 +87,32 @@ _handleChange = (e) => {
         <System.P>Declare the CardTabGroup component.</System.P>
         <br />
         <System.CodeBlock>
-          {`<System.CardTabGroup
-  name="eighteen"
-  options={TAB_GROUP_TWO}
-  value={this.state.eighteen}
-  onChange={this._handleChange}
-/>
+          {`
+const CardTabGroupOne = () => {
+   return (
+      <div>
+        <CardTabGroup
+          name="eighteen"
+          options={TAB_GROUP_TWO}
+          value={this.state.eighteen}
```

Comment:
These state values are bad, they are my fault, would you be willing to help me change the names of the states to something more meaningful?